components/LocationComparisonSection/LocationTagsList.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FunctionComponent } from 'react';
+
+const LocationTagsList: FunctionComponent = ({ children }) => {
+  return <ul className="m-pills">{children}</ul>;
+};
+
+export { LocationTagsList };
```

Comment:
This seems like one of those widgets we're likely to reuse some day ... so please move this (& all related components) outside of the LocationComparisonSection into it's own folder, and change it's name to something that doesn't have `location` in it... it could be showing tags of food next time we use it.

Reply:
You'll need to add unit tests for them all.